blog/book-review-the-goal-a-process-of-ongoing-improvement.org

```diff
@@ -0,0 +1,229 @@
+- inventory ::
+  This one is tricky.
+
+  I think inventory in software development as the features that are under development and are still not shipped to users.
+  And you can see inventory at every step of software development:
+  1. Feature requirements / specifications that are not runnable code yet.
+  2. Design mockups / wireframes that are not usable at all.
+  3. Code that are not merged nor deployed to production.
+  As the book explains, *when inventory goes up, the carrying costs goes up*.
+  The more feature branches you have, the more integration costs you need to pay.
+  The more features you are working on, the more possible to ship something that is not working as expected.
+  This is why we need to limit WIP. Because it helps reduce our inventory.
+
+  But the question then becomes how to measure the value of these WIP features?
+  It's a hard question and the answer varies from product to product.
+  (We may use how much a user want to pay for it and how many users may pay for it to estimate the value of a WIP feature.)
+- operational expense ::
+  Everything else is operational expense.
+  + Labor costs.
+  + Tools we are using (Laptops, editors, IDEs, software licenses, etc).
+  + The server that our code runs on.
+
+Only by understanding these 3 key metrics, can we start identifying the bottleneck in our feature development pipeline.
+** How to making more money as an individual?
+We can apply the same thinking to an individual as well:
+If our goal as an individual is to make money, then what's our throughput, inventory, and operational expense?
+- throughput ::
+  If you are like me, having a stable work and income, our throughput is very stable and predictable: salary, compensations, etc.
+- inventory ::
+  Then, what is our inventory?
+  What's the money currently inside us?
+
+  If we think at a higher level, our inventory is the knowledge we have.
+  Every piece of knowledge we have is value and can be transformed into throughput.
+
+  Coming down to a lower level, our inventory is the projects we are building.
+  This blog you're reading now is one of my inventory.
+  My GitHub account and the various projects hosted on it are my inventories.
+  These are the outcome of my knowledge and can be turned into throughput.
+- operational expense ::
+  For a human being, the operational expense is the cost that keep us alive.
+  Expense spent on food, house, clothes, etc.
+
+  And for us to be able to transform our inventories into throughput, our operational expense might include money we spent on our tools.
+
+So, to make more money as an individual, the main focus is to turn as many inventories into throughput.
+- Increase salary by delivering more values with your knowledge.
+- Transform more knowledge into purchasable products.
+
+And think about your bottleneck in this process.
+(My guess is that for most of us, the bottleneck is marketing/sales, i.e. let potential buyers know that you have the knowledge they need.)
```

Comment:
This is actually very similar to the leverage theory in the Effective Engineer, just from a different angle